### PR TITLE
test Utils class

### PR DESCRIPTION
**Pull request by Andati**

### CODE REVIEW

File: opensrp-child/src/test/java/org/smartregister/child/util/UtilsTest.java [hunk 1]
```diff
@@ -121,4 +131,57 @@ public void testFormatAtBirthKey() {
...
+    public void testContext() {
+        PowerMockito.mockStatic(ChildLibrary.class);
+        PowerMockito.when(ChildLibrary.getInstance()).thenReturn(childLibrary);
+        Context context = Utils.context();
+        Assert.assertEquals(childLibrary.context(), context);
+    }
+
+    @Test
+    public void testGetWeeksDue() {
+        int i = Utils.getWeeksDue(new DateTime());
```
Comment (ndegwamartin) [inline opensrp-child/src/test/java/org/smartregister/child/util/UtilsTest.java:165]: I think we need to cover more edge cases here

File: opensrp-child/src/test/java/org/smartregister/child/util/UtilsTest.java [hunk 2]
```diff
@@ -121,4 +131,57 @@ public void testFormatAtBirthKey() {
...
+    @Test
+    public void testGetDataRow() {
+        TableRow tr = Utils.getDataRow(context);
+        Assert.assertEquals(0, tr.getPaddingTop());
+    }
+
+    @Test
+    public void testGetDataRow2() {
+        TableRow tr = Utils.getDataRow(context, "label", "value", null);
+        Assert.assertEquals(0, tr.getPaddingTop());
```
Comment (ndegwamartin) [inline opensrp-child/src/test/java/org/smartregister/child/util/UtilsTest.java:179]: We can add more assertions for the various text views expected to be created by this method for the row

File: opensrp-child/src/test/java/org/smartregister/child/util/UtilsTest.java [hunk 3]
```diff
@@ -121,4 +131,57 @@ public void testFormatAtBirthKey() {
...
+    }
+
+    @Test
+    public void testGetDataRow3() {
+        TableRow tr = Utils.getDataRow(context, "label", "value", "field", null);
+        Assert.assertEquals(0, tr.getPaddingTop());
```
Comment (ndegwamartin) [inline opensrp-child/src/test/java/org/smartregister/child/util/UtilsTest.java:185]: We can add more assertions for the various text views expected to be created by this method for the row
